src/main/java/com/algorand/algosdk/builder/transaction/ApplicationBaseTransactionBuilder.java

```diff
@@ -103,10 +103,6 @@ private List<Transaction.BoxReference> convertBoxes(List<AppBoxReference> abrs,
         return xs;
     }
 
-    /**
-     * BoxReferences lists the boxes whose state may be accessed during the execution
-     * of this application call. The access is read-only.
-     */
```

Comment:
By deleting this comment, we'll inherit the comment from `ApplicationCallReferencesSetter`